middleware/session/store.go

```diff
@@ -0,0 +1,60 @@
+package session
+
+import (
+	"fmt"
+
+	"github.com/gofiber/fiber/v2"
+)
+
+type Store struct {
+	Config
+}
+
+func New(config ...Config) *Store {
+	// Set default config
+	cfg := configDefault(config...)
+
+	return &Store{
+		cfg,
+	}
+}
+
+func (s *Store) Get(c *fiber.Ctx) *Session {
```

Comment:
looking at this signature, it doesn't feel right to me, as there's no way to know if there was an error or not, right? so you could rely on some security-related stuff to be in the session, but you could get get a fresh session instead of old one, which could end up in lost session-wide parameters, "ghost sessions", failed aug PAKE auth etc.

Reply:
Fixed in <https://github.com/gofiber/fiber/pull/1009/commits/2cafd486ef3b821adb64385b3a0d95f6f8488e7f> thanks